blast_config/command_list.c

```diff
@@ -227,10 +239,262 @@ struct mcom mcommands[plugh + 2] = {
 
     /* DETECTORS */
 
+    /* NEW STAR CAMERAS */
+    // SC1
+    {COMMAND(sc1_trim_lat), "Send the commanded Latitude to SC1", GR_XSC_PARAM, 1,
+        {
+            {"Latitude", -90., 90., 'f', "NONE"}
+        }
+    },
+    {COMMAND(sc1_trim_lon), "Send the commanded Longitude to SC1", GR_XSC_PARAM, 1,
+        {
+            {"Longitude", -180., 180., 'f', "NONE"}
+        }
+    },
+    {COMMAND(sc1_trim_height), "Send the commanded Altitude to SC1", GR_XSC_PARAM, 1,
+        {
+            {"Altitude", 0., 50000., 'f', "NONE"}
+        }
+    },
+    {COMMAND(sc1_trim_pos), "Send the commanded position to SC1", GR_XSC_PARAM, 3,
+        {
+            {"Latitude", -90., 90., 'f', "NONE"},
+            {"Longitude", -180., 180., 'f', "NONE"},
+            {"Altitude", 0., 50000., 'f', "NONE"}
+        }
+    },
+    {COMMAND(sc1_set_exposure_time), "Set SC1 exposure time (msec)", GR_XSC_PARAM, 1,
+        {
+            {"Exposure time (msec)", 10., 1000., 'f', "NONE"}
+        }
+    },
+    {COMMAND(sc1_set_logodds), "Set SC1 astrometry logodds", GR_XSC_PARAM, 1,
+        {
+            {"Logodds", 0., 10000000000., 'f', "NONE"}
+        }
+    },
+    {COMMAND(sc1_set_time_limit), "Set the number of solve attempts SC1 is limited to per photo", GR_XSC_PARAM, 1,
```

Comment:
Fix description, this is a time limit in seconds rather than number of attempts, right?

Reply:
Nope, still the number of iterations (time length is not deterministic and interrupts didn't work well with 1 API call holding stuff up)